SECP-service/src/main/resources/assets/app/views/profile/user-profile.html

```diff
@@ -0,0 +1,35 @@
+<div>
+    <div>
+        <div id="editbutton" ng-show="EditProfileAvailable()">
```

Comment:
shouldn't there be a button here. Nothing is showing on the page.

Reply:
Thats cuz theres no username ltahir in ur localstorage. Add variable username with key ltahir and u should see it then

Reply:
sounds good.